QUEST_LV_0100.tsv

```diff
@@ -1281,7 +1281,7 @@ QUEST_LV_0100_20150317_001276	The reason why this isolated canyon is beautiful i
 QUEST_LV_0100_20150317_001277	The first mission of our family has been just achieved.{nl}This is just the start.
 QUEST_LV_0100_20150317_001278	Supply Commander Vio
 QUEST_LV_0100_20150317_001279	Today is quiet.{nl}I am talking about those monsters.
-QUEST_LV_0100_20150317_001280	We are veterans.{nl}After all those years as a mercenary, we should not be confused even if we haven't heard anything in years.
+QUEST_LV_0100_20150317_001280	We are veterans.{nl}We should not panic even if we are cut off from society and haven't heard anything back in a few years.
```

Comment:
We should not panic even... -> Living as a mercenary these few years, we should not panic even